data/org.freedesktop.impl.portal.AppChooser.xml

```diff
@@ -62,6 +64,14 @@
             <listitem><para>The filename to choose an application for. Note that this
                is just a basename, without a path.</para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>activation_token s</term>
+            <listitem><para>
+              A token that can be used to activate the file chooser.
```

Comment:
Is it a typo that this talks about a file chooser ?

Reply:
Ah, indeed, "file chooser" is wrong here. But as far as I understand, since this is the backend API, this D-Bus call will just present the file chooser to the user, and won't actually start it.

I've updated this option to refer to "the application chooser" instead.

And I've added an activation token in results as well, which was missing. This is necessary in case choosing an application has required user interaction: the portal impl should generate a new activation token from that interaction.